src/axom/spin/docs/sphinx/octree.rst

```diff
@@ -0,0 +1,73 @@
+SpatialOctree
+^^^^^^^^^^^^^
+
+Axom provides an implementation of the octree spatial index.  The
+``SpatialOctree`` recursively divides a bounding box into a hierarchy of
+non-intersecting bounding boxes.  Each level of subdivision divides the bounding
+box of interest along each of its dimensions, so 2D ``SpatialOctree`` objects
+contain four child bounding boxes at each level, while 3D objects contain eight
+children at each level.  The process stops when each a bounding box contains
```

Comment:
> The process stops when....

This sentence is not strictly true.  The `SpatialOctree` (through is base class `OctreeBase`) provides a means to refine a leaf block, but neither of these classes define when this refinement should happen.

See: https://github.com/LLNL/axom/blob/d23ed293165c16f58a301ba2f387efd527416c8d/src/axom/spin/OctreeBase.hpp#L817-L823

Reply:
It might be best to remove this sentence.

The sentence that you wrote below is accurate:

> Unlike the ``BVHTree`` class, the ``SpatialOctree`` is intended as a building
block for further specialization.  Please see the ``quest::InOutOctree`` as an
example of this.

Reply:
I removed the sentence "The process stops..."